pkgs/by-name/pu/pushlog/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "pushlog";
+  version = "0.3.1";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "serpent213";
+    repo = "pushlog";
+    rev = "v${version}";
+    sha256 = "sha256:1cfag0vfi07vnww5sls630xqc41rnyaqzrabia0fwhiill3k6dfm";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    click
+    fuzzywuzzy
+    levenshtein
+    pyyaml
+    setuptools
+    systemd
+  ];
```

Python dependencies use the dependencies key, while build dependencies use build-dependencies

```suggestion
  build-dependencies = with python3Packages; [
    setuptools
  ];

  dependencies = with python3Packages; [
    click
    fuzzywuzzy
    levenshtein
    pyyaml
    systemd
  ];
```

---

That's the one thing I could not get to work, so I went with:
```nix
  nativeBuildInputs = with python3Packages; [
    setuptools
  ];

  dependencies = with python3Packages; [
    click
    fuzzywuzzy
    levenshtein
    pyyaml
    systemd
  ];
```